pkg/k8shandler/deployment.go

```diff
@@ -32,7 +32,7 @@ func (node *deploymentNode) isDifferent(cfg *desiredNodeState) (bool, error) {
 	for _, container := range node.resource.Spec.Template.Spec.Containers {
 		if container.Name == "elasticsearch" {
 			if container.Image != cfg.ESNodeSpec.Spec.Image {
-				logrus.Debugf("Container image '%v' is different that desired, updating..", container.Image)
+				logrus.Infof("Container image '%v' for node '%s' is different that desired, updating..", container.Image, node.resource.Name)
```

Comment:
We only have one container now. 
What about `("Elasticsearch image change found, updating %q", node.resource.Name)`

Reply:
so...
`"Container image '%v' is different that desired"`